src/main/java/WayofTime/bloodmagic/api/util/helper/ItemHelper.java

```diff
@@ -0,0 +1,149 @@
+package WayofTime.bloodmagic.api.util.helper;
+
+import WayofTime.bloodmagic.api.altar.IBloodAltar;
+import WayofTime.bloodmagic.api.iface.IFillable;
+import WayofTime.bloodmagic.api.iface.IUpgradeTrainer;
+import WayofTime.bloodmagic.api.livingArmour.LivingArmourHandler;
+import WayofTime.bloodmagic.api.livingArmour.LivingArmourUpgrade;
+import WayofTime.bloodmagic.item.ItemUpgradeTome;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+public class ItemHelper
```

Comment:
This class is doing too much, I feel - why should I use the ItemHelper, which is a generic class, when I want to change a specific item (the upgrade tomes)?